### PR TITLE
Move serving samples to index.md

### PR DESCRIPTION
**Pull request by evankanderson**
Remove `README.md` file includes for docs/serving/samples. I'm breaking these up into several PRs to make it easier to avoid conflicts / hold off on PRs that would have a conflict.

## Proposed Changes 

- Inline `README.md` files that are included via `{{readfile}}` shortcodes.

### CODE REVIEW

**Comment by evankanderson** [comment]
/approve
/assign @julz @abrennan89 

(I'm assuming this makes `mkdocs` migration no harder and may simplify things overall.)

**Comment by julz** [comment]
Yep, I currently have excludes set up in mkdocs.yml to hide all the index files with shortcodes, which this would let us drop, so \o/.

I'll let someone with more docs context lgtm, but lgtm :).

File: docs/serving/samples/cloudevents/cloudevents-dotnet/index.md [hunk 1]
```diff
@@ -5,4 +5,114 @@ weight: 1
 type: "docs"
 ---
 
-{{% readfile file="README.md" %}}
```
Comment (abrennan89) [inline docs/serving/samples/cloudevents/cloudevents-dotnet/index.md:8]: I think the filename needs to be `_index.md` like with the others?

File: docs/serving/samples/cloudevents/cloudevents-nodejs/index.md [hunk 2]
```diff
@@ -5,4 +5,131 @@ weight: 1
 type: "docs"
 ---
 
-{{% readfile file="README.md" %}}
```
Comment (abrennan89) [inline docs/serving/samples/cloudevents/cloudevents-nodejs/index.md:8]: same thing re filename 🙂

**Review by abrennan89 — CHANGES_REQUESTED** [review]
I think the filenames should be `_index.md` but not sure? A lot of them seem to be just `index.md`?

**Comment by evankanderson** [comment]
Furthermore, an existence proof that these don't need to be `_index.md` is that the current site renders okay. 😁

**Review by abrennan89 — APPROVED** [review]
Ah, I got you 😄 in that case...

/lgtm
/approve

Thanks! 🙂

**Pull request merged**